### PR TITLE
feat: implemented settings page function

### PR DESCRIPTION
**Pull request by himanshukesarwani-dev**
- implemented the settings page function
- takes a as argument.

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -94,4 +94,8 @@ function evenOdd(num) {
 
 function profilePage(img){
     return "image"
+}
+
```
Comment (jayshreejaiswal) [inline math.js:98]: can you please add documentation

**Review by jayshreejaiswal — CHANGES_REQUESTED** [review]

File: math.js (hunk 1 above)
Comment (himanshukesarwani-dev) [reply math.js:98]: added the documentation.

File: math.js [hunk 2]
```diff
@@ -96,6 +96,13 @@ function profilePage(img){
     return "image"
 }
 
+
+/**
+ * settings page function documentation.
+ * @param {number} a
```
Comment (jayshreejaiswal) [inline math.js:102]: can you please correct your commit meassage

**Review by jayshreejaiswal — CHANGES_REQUESTED** [review]

File: math.js (hunk 2 above)
Comment (himanshukesarwani-dev) [reply math.js:102]: sorry, added the correct commit message.

**Pull request merged**